pytripgui/canvas_vc/gui_state.py

```diff
@@ -0,0 +1,15 @@
+from typing import Optional, List
+
+from pytripgui.canvas_vc.projection_selector import ProjectionSelector
+
+
+class PatientGuiState:
+    """
+    This class holds information about viewing patient
+        - on which slice user stopped
```

Comment:
on which slice which action was stopped ?

Reply:
you mean that `scrolling` is missing?
`on which slice user stopped scrolling`
even better
`on which slice user stopped scrolling on each plane`

Reply:
last one is best

Reply:
Done